push-v2.md

```diff
@@ -1088,6 +1088,26 @@ Example:
 ]
 ```
 
+####  service_reminder_lead
+
+|   Name   |   Type   |  Unit/Format        | Example                  |                   Description                   |
+|:--------:|:--------:|:-------------------:|--------------------------|-------------------------------------------------|
+| value | datetime  | RFC 3339          | 2022-05-19T18:31:03.000Z       | Service date
+
+
+Example:
+
+``` json
+[
+    {
+        "type": "service_reminder_lead",
+        "id": 12095712,
+        "vehicleId": 59821,
+        "time": "2022-01-12T07:06:45",
+        "value": "2022-05-19T18:31:03"
```

Comment:
The example above doesn't match the example down here. Also why does this have the time part? Shouldn't it just be a date?

Reply:
Good point. I will update

Reply:
Couldn't we just use `2022-05-19` instead of `2022-05-19T00:00:00.000Z`?

Reply:
Alright, then we'll just stick with that.